backend/graph/dummy.go

```diff
@@ -11,9 +25,9 @@ var dummyProcesses = []*model.Process{
 		DeploymentTime:       "2023-01-01T00:00:00Z",
-		Instances:            []*model.Instance{},
+		Instances:            dummyInstances,
```

Comment:
FYI, this could empty array like before since it will be ignored, and instances will queried from the process specific resolver you created. It's up to you if you want to change this but it won't matter in the end.